internal/bminventory/inventory.go

```diff
@@ -2319,6 +2319,13 @@ func (b *bareMetalInventory) updateClusterInternal(ctx context.Context, v1Params
 		return nil, err
 	}
 
+	err = b.updateClusterNetworkVMUsage(cluster, usages)
+	if err != nil {
+		log.WithError(err).Error("Failed to update usage flag for 'User managed network with VMs'.")
```

Comment:
better log inside the function and leave it in the same format as other functions

Reply:
👍🏽